core/src/main/java/org/apache/brooklyn/core/entity/Attributes.java

```diff
@@ -52,11 +52,15 @@
 
     // TODO these should switch to being TemplatedStringAttributeSensorAndConfigKey
     BasicAttributeSensorAndConfigKey<String> DOWNLOAD_URL = new BasicAttributeSensorAndConfigKey<String>(
-            String.class, "download.url", "URL pattern for downloading the installer (will substitute things like ${version} automatically)");
+            String.class, 
+            "download.url", 
+            "URL pattern for downloading the installer (will substitute things like ${version} automatically)");
```

Comment:
Shouldn't it just say `uses FreeMarker templating format` to be consistent with the other description below?